config/locales/pt-BR.yml

```diff
@@ -314,7 +314,7 @@ pt-BR:
       premade2_description: 'Se sentindo otimista com relação a alcançar metas'
       premade3_name: Curiosidade
       premade3_description: 'Sentir curiosidade sobre fatos incertos.'
-      premade4_name: 'Sentir-se Ressentido'
+      premade4_name: 'Sentir-se ressentido'
       premade4_description: 'Não aceitar certos fatos, sentir raiva, sentimento de vingança, e/ou ciúmes.'
```

Comment:
remoção da vírgula depois de 'vingança', não sei se isso está 'errado' apenas uma dica xD

Reply:
Passou batido! Eu to removendo essas vírgulas mesmo haha